.github/workflows/main.yml

```diff
@@ -87,7 +87,7 @@ jobs:
       run: ./run_all_tests.sh
       env:
         EXASOL_VERSION: "${{ matrix.exasol_version }}"
-      if: github.ref == 'refs/heads/master' || contains(github.event.head_commit.message, '[run all tests]') || github.event_name != 'push'
+      if: github.ref == 'refs/heads/master' || contains(github.event.head_commit.message, '[run all tests]')
```

Comment:
That's interesting. Maybe this is an info for the PullRequest template (otherwise it will get lost):
...
If you want to run all test, remember to add "run all tests" in the commit msg....
....

Reply:
yes, I also thought about, if this is repo might be a candidate for a develop branch. Such, that you can run all tests only for the last exasol version in PR and in the develop branch, then for all. Or, alternative, we could have it as an extra workflow which needs to run for the merge, but doesn't run automatically.

Reply:
I talked to @jakobbraun about that, he suggested creating an environment with approval and use this environment in the workflow where we run all tests for all versions